src/lib/country-data.js

```diff
@@ -12,6 +12,32 @@ module.exports.countryOptions = Object.keys(countries).map(code => ({
         return 1;
     });
 
+const overrideOptions = module.exports.overrideOptions = (countryNameOverrides, thisCountryOptions) => {
+    if (!thisCountryOptions) thisCountryOptions = countryOptions;
+    // shallow copy two levels deep, since that's as low as countryOptions goes
+    const newCountryOptions = thisCountryOptions.map(item => ({...item}));
+    Object.keys(countryNameOverrides).forEach(codeToOverride => {
+        const existing = newCountryOptions
+            .find(countryOption => countryOption.value === codeToOverride.toLowerCase());
+        if (existing) existing.label = countryNameOverrides[codeToOverride];
+    });
+    return newCountryOptions;
+};
+
+const commonOptions = module.exports.commonOptions = (commonCountryCodes, thisCountryOptions) => {
+    if (!thisCountryOptions) thisCountryOptions = countryOptions;
+    const newCountryOptions = thisCountryOptions.map(item => ({...item}));
+    commonCountryCodes.reverse().forEach(commonCountryCode => {
+        const existing = newCountryOptions
+            .find(countryOption => countryOption.value === commonCountryCode.toLowerCase());
+        if (existing) newCountryOptions.unshift(existing);
+    });
+    return newCountryOptions;
+};
+
+module.exports.processedCountryOptions =
```

Comment:
Same here. What does "processed" mean? Looks like the code does something very specific, but has a very generic name.

Reply:
yeah, i'm not sure what to call this. it's like, the version of country options that we've messed with so it's what we want. ideas?

Reply:
I'm trying "registrationCountryOptions"